stan/math/rev/fun/from_var_value.hpp

```diff
@@ -25,6 +25,45 @@ Eigen::Matrix<var, T::RowsAtCompileTime, T::ColsAtCompileTime> from_var_value(
   return res;
 }
 
+/**
+ * Forward Eigen containers of vars.
+ *
+ * @tparam T type of the input
+ * @param a matrix to convert
+ */
+template <typename T, require_eigen_vt<is_var, T>* = nullptr>
+auto from_var_value(T&& a) {
```

Comment:
```suggestion
T from_var_value(T&& a) {
```
Picking up bad habits from Steve? No reason for this to be auto if you know the type.

Reply:
You just like it cuz it's a T.